### PR TITLE
Better identify

### PR DESCRIPTION
**Pull request by hwang-pku**
change for using recursive search for the test class. 
Also fix some minor bugs.

However, we still need to find out how to deal with dependency issue.

### CODE REVIEW

File: confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/ConfigurationDebugNewJvmMojo.java [hunk 1]
```diff
@@ -316,6 +316,7 @@ private void init(CampaignConfiguration configuration) throws IOException {
         List<String> javaOptions = new ArrayList(configuration.getJavaOptions());
         javaOptions.add("-cp");
         String classPath = configuration.getTestClasspathJar().getAbsolutePath() + File.pathSeparator + this.debugFrameworkJar.getAbsolutePath();
+        System.out.println(classPath);
```
Comment (shuaiwang516) [inline confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/ConfigurationDebugNewJvmMojo.java:319]: Can you remove the print?

File: confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/IdentifyCTestMojo.java [hunk 2]
```diff
@@ -160,7 +165,7 @@ public JvmLauncher setupJVM(String testClass, String testMethod) throws MojoExec
             javaOptions.add("-Dclass=" + testClass);
             javaOptions.add("-Dmethod=" + testMethod);
             javaOptions.add("-cp");
-            String classPath = System.getProperty("java.class.path");
+            String classPath = System.getProperty("user.home") + "/.m2/repository/edu/illinois/confuzz/confuzz-meringue-extension/1.0.0-SNAPSHOT/confuzz-meringue-extension-1.0.0-SNAPSHOT.jar";
```
Comment (shuaiwang516) [inline confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/IdentifyCTestMojo.java:168]: Why do we need to hardcode the CP?
Isn't this a maven dependency and should be included in `project.getTestClasspathElements()`?

Comment (hwang-pku) [reply confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/IdentifyCTestMojo.java:168]: We need to find Maven-meringue-plugin. I'm not sure why fuzz goal works but identify not.

File: confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/ConfigurationDebugNewJvmMojo.java (hunk 1 above)
Comment (hwang-pku) [reply confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/ConfigurationDebugNewJvmMojo.java]: Resolved in the latest commit.

File: confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/IdentifyCTestMojo.java [hunk 3]
```diff
@@ -165,7 +166,7 @@ public JvmLauncher setupJVM(String testClass, String testMethod) throws MojoExec
             javaOptions.add("-Dclass=" + testClass);
             javaOptions.add("-Dmethod=" + testMethod);
             javaOptions.add("-cp");
-            String classPath = System.getProperty("user.home") + "/.m2/repository/edu/illinois/confuzz/confuzz-meringue-extension/1.0.0-SNAPSHOT/confuzz-meringue-extension-1.0.0-SNAPSHOT.jar";
+            String classPath = System.getProperty("java.class.path") + ":" + FileUtil.getClassPathElement(IdentifyForkMain.class).getAbsolutePath();
```
Comment (shuaiwang516) [inline confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/IdentifyCTestMojo.java:169]: @hwang-pku I rewrite your hardcode jar here.

Comment (hwang-pku) [reply confuzz-maven-plugin/src/main/java/edu/illinois/confuzz/IdentifyCTestMojo.java:169]: I see. LGTM!

**Pull request merged**